src/translations/en.json

```diff
@@ -2678,19 +2678,19 @@
             "encryption": {
               "title": "Encryption",
               "description": "All your backups are encrypted by default to keep your data private and secure.",
-              "location_encrypted": "This location is encrypted",
-              "location_unencrypted": "This location is unencrypted",
-              "location_encrypted_description": "Your data private and secure by securing it with your encryption key.",
+              "location_encrypted": "This location contains encrypted backups",
```

Comment:
I don't think this is the correct phrase here, something like `Backups made to this location will be encrypted`